### ISSUE TITLE
New installation fails testing w/ AttributeError: 'module' object has no attribute 'nest_codegenerator_test'

### ISSUE THREAD

**Issue by harryHoward** [nest/nestml#424]
I have tried to install NestML on a working NEST 2.16.0 installation on a brand new MacBook Pro under OSX 10.13.6 with the following Python interpreter:

```
Python 2.7.15 |Anaconda, Inc.| (default, May  1 2018, 18:37:05) 
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
```
I installed from the Terminal by downloading & decompressing the folder (nestml-master) from https://github.com/nest/nestml and then:

```
$ cd /Users/me/nestml-master
$ python setup.py install --user
```

This worked with no error messages. I can import NestML in Python without an error:

`>>> import pynestml`

However, if I test the installation, it fails:

```
$ python setup.py test
… AttributeError: 'module' object has no attribute 'nest_codegenerator_test'
```

Yet nest_codegenerator_test.py is definitely in the build folder.

Also,  if I try to invoke NestML directly from the command line, it also fails:

```
$ python PyNestML.py -h
Traceback (most recent call last):
  File "PyNestML.py", line 22, in <module>
    from pynestml.frontend.pynestml_frontend import main
  File "/Users/me/nestml-master/pynestml/frontend/pynestml_frontend.py", line 25, in <module>
    from pynestml.codegeneration.nest_codegeneration import analyse_and_generate_neurons, generate_nest_module_code
  File "/Users/me/nestml-master/pynestml/codegeneration/nest_codegeneration.py", line 25, in <module>
    from odetoolbox import analysis
ImportError: No module named odetoolbox
```
Any help would be greatly appreciated,
Harry

**Comment by harryHoward** [comment]
After looking at the error message again, it occurred to me to see if odetoolbox was in my Python installation. It was not. I found its website (a part of NEST, but for some reason it was not included in the source file) and downloaded and installed it. I still got an error, because it depends on pygsl, which I installed with pip. 

So in summary, if I had to do it over again, this would be the way to proceed, since I use Miniconda and prefer to install everything into the conda directory with pip (and simpy includes antlr):

$ conda install simpy, numpy
$ pip install pygsl
(download ode toolbox source from https://github.com/nest/ode-toolbox)
$ pip install /path_to/ode-toolbox-master
(download NestML source from https://github.com/nest/nestml)
$ pip install /path_to/nestml-master
$ cd /path_to/nestml-master
$ python setup.py test
(this took 779 seconds)
(test again by trying to load a crucial component from NestML)
$ python from pynestml.frontend.pynestml_frontend import to_nest,install_nest

Now to finally start doing some modeling.
Harry
